lib/utils.js

```diff
@@ -77,16 +77,34 @@ function waitForAvailable(host, port, options, callback) {
       count--;
       if (available) return callback();
       if (count === 0) return callback(new Error('Server is unavailable'));
-      setTimeout(() => run(), options.retryMs);
+      setTimeout(() => run(), options.retryMS);
     });
   }
 
   run();
 }
 
+function createCommandOptions(manager, localOptions) {
+  const options = clone(manager.clientOptions);
```

Comment:
Also FWIW I think most of this function can just be an `Object.assign` right now:
```
const managerOptions = manager.options || {};
const options = Object.assign({}, manager.clientOptions, {
  host: typeof managerOptions.host === 'string' ? managerOptions.host : 'localhost',
  port: typeof managerOptions.port === 'number' ? managerOptions.port : '27017',
  .. the rest of them ..
});
```

Reply:
👍 Only change was I removed `managerOptions` since I don't see any reason why `manager.options` should **not** be defined. It is defaulted to {} in the constructor.